packages/atlas/src/components/_inputs/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,46 @@
+import styled from '@emotion/styled'
+import React from 'react'
+
+import { RadioButton, RadioButtonProps } from '@/components/_inputs/RadioButton'
+import { sizes } from '@/styles'
+
+export type RadioButtonGroupProps = {
+  options: RadioButtonProps[]
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+  value: string | number
+  name?: string
+  disabled?: boolean
+  error?: boolean
+}
+
+export const RadioButtonGroup: React.FC<RadioButtonGroupProps> = ({
```

Comment:
Can we actually make use of this component and see it in action? We could apply this in the VideoForm at least?

Reply:
This was very good idea 👍